server/views/pages/staff-dashboard.njk

```diff
@@ -8,9 +8,7 @@
   <div class="filters govuk-!-margin-bottom-4" id="prisonersFilter">
     <form action="/#prisonersFilter" method="GET" novalidate>
       <div class="govuk-grid-row">
-        {% if myCasesEnabled %}
-          <div class="govuk-grid-column-three-quarters">
-        {% endif %}
+        <div {% if myCasesEnabled %} class="govuk-grid-column-three-quarters" {% else %} class="govuk-grid-column-full" {% endif %}>
```

Comment:
you can write this like `class="{{ "govuk-grid-column-three-quarters" if myCasesEnabled else "govuk-grid-column-full" }}"`